metrics.md

```diff
@@ -0,0 +1,238 @@
+    sum(dels) as dels,
+    sum(aoks) as aoks,
+    sum(oops) as oops,
+    sum(fups) as fups,
+    sum(r_clients) as r_clients,
+    sum(r_crypto) as r_crypto,
+    sum(r_forms) as r_forms,
+    sum(r_history) as r_history,
+    sum(r_keys) as r_keys,
+    sum(r_meta) as r_meta,
+    sum(r_bookmarks) as r_bookmarks,
+    sum(r_prefs) as r_prefs,
+    sum(r_tabs) as r_tabs,
+    sum(r_passwords) as r_passwords,
+    sum(r_addons) as r_addons,
+    sum(w_clients) as w_clients,
+    sum(w_crypto) as w_crypto,
+    sum(w_forms) as w_forms,
+    sum(w_history) as w_history,
+    sum(w_keys) as w_keys,
+    sum(w_meta) as w_meta,
+    sum(w_bookmarks) as w_bookmarks,
+    sum(w_prefs) as w_prefs,
+    sum(w_tabs) as w_tabs,
+    sum(w_passwords) as w_passwords,
+    sum(w_addons) as w_addons
+  from
+  (select
+    uid,
+    dev,
+    ts,
+    t,
+    decode(substring(ua_os,0,8), 'iPad', 'ios', 'iPod', 'ios', 'iPhone', 'ios', 'Android', 'android', 'Windows', 'windows', 'Macinto', 'mac', 'Linux', 'linux', null, 'unknown', 'other') as ua_os,
+    ua_browser,
+    ua_version,
+    case when bucket = 'clients' and method = 'GET' then 1 end as r_clients,
+    case when bucket = 'crypto' and method = 'GET' then 1 end as r_crypto,
+    case when bucket = 'forms' and method = 'GET' then 1 end as r_forms,
+    case when bucket = 'history' and method = 'GET' then 1 end as r_history,
+    case when bucket = 'keys' and method = 'GET' then 1 end as r_keys,
+    case when bucket = 'meta' and method = 'GET' then 1 end as r_meta,
+    case when bucket = 'bookmarks' and method = 'GET' then 1 end as r_bookmarks,
+    case when bucket = 'prefs' and method = 'GET' then 1 end as r_prefs,
+    case when bucket = 'tabs' and method = 'GET' then 1 end as r_tabs,
+    case when bucket = 'passwords' and method = 'GET' then 1 end as r_passwords,
+    case when bucket = 'addons' and method = 'GET' then 1 end as r_addons,
+    case when bucket = 'clients' and method = 'POST' then 1 end as w_clients,
+    case when bucket = 'crypto' and method = 'POST' then 1 end as w_crypto,
+    case when bucket = 'forms' and method = 'POST' then 1 end as w_forms,
+    case when bucket = 'history' and method = 'POST' then 1 end as w_history,
+    case when bucket = 'keys' and method = 'POST' then 1 end as w_keys,
+    case when bucket = 'meta' and method = 'POST' then 1 end as w_meta,
+    case when bucket = 'bookmarks' and method = 'POST' then 1 end as w_bookmarks,
+    case when bucket = 'prefs' and method = 'POST' then 1 end as w_prefs,
+    case when bucket = 'tabs' and method = 'POST' then 1 end as w_tabs,
+    case when bucket = 'passwords' and method = 'POST' then 1 end as w_passwords,
+    case when bucket = 'addons' and method = 'POST' then 1 end as w_addons r_history,
+  from sync0715)
+group by uid, dev);
+```
+
+### Device Counts
+
+This table summarizes how many active devices each user had on a given day, where an active device is one that has had any activity in the last 7 days.
```

Comment:
hey @davismtl, this one matches your request for "activity in last 7 days", so we should definitely go from 5 to 7 in FxA for consistency.